kolibri/core/assets/src/core_app_mediator.js

```diff
@@ -44,6 +45,35 @@ module.exports = class Mediator {
     this._asyncCallbackRegistry = {};
 
     this._eventDispatcher = new Vue();
+
+    /**
```

Comment:
Not too opinionated, but wondering if this should happen in the Facade constructor, rather than the mediator?